server/datastore/memorydb/index.ts

```diff
@@ -58,4 +58,12 @@ export class InMemoryDatastore implements Datastore {
     }
     this.posts.splice(index, 1);
   }
+
+  getLikes(postId: string): number {
+    return this.likes.filter(x => x.postId == postId).length;
```

Comment:
nitpick: [prefer strict equality](https://developer.mozilla.org/en-US/docs/web/javascript/equality_comparisons_and_sameness) using triple equals in javascript: `x => x.postId === postId`, which is safer.